lib/SIL/Utils/SILBridging.cpp

```diff
@@ -402,11 +402,6 @@ static_assert((int)BridgedMemoryBehavior::MayWrite == (int)swift::MemoryBehavior
 static_assert((int)BridgedMemoryBehavior::MayReadWrite == (int)swift::MemoryBehavior::MayReadWrite);
 static_assert((int)BridgedMemoryBehavior::MayHaveSideEffects == (int)swift::MemoryBehavior::MayHaveSideEffects);
 
-static_assert((int)BridgedInstruction::AccessKind::Init == (int)swift::SILAccessKind::Init);
```

Comment:
nit: can we keep the static_assert that c++ enum values are same as swift ones ?

Reply:
> nit: can we keep the static_assert that c++ enum values are same as swift ones ?

That would be great. Can you tell me how to do it?

Reply:
I think it should work if you keep the original `AccessKind` in `SILBridging.h` and then create the swift enum this way: 

```
public var accessKind: AccessKind {
   AccessKind(rawValue: bridged.BeginAccessInst_getAccessKind().rawValue)!
 }
```

Reply:
Huh, I guess that's not sufficient either to truly ensure these enum values  will be equivalent.

Reply:
And way too complex and costly. So please ignore my comment!

Reply:
Maybe add a comment for `SILAccessKind` and `BeginAccessInst.AccessKind` that both must be kept in sync